src/types/markers.js

```diff
@@ -176,6 +176,8 @@ export type GCMinorCompletedData = {
 
   // The size of the data moved into the tenured heap.
   bytes_tenured: number,
+  // The number of cells tenured (not in older profiles).
```

Comment:
instead of `not in older profiles`, more useful would be a link to the bug number that introduced this.